powermon/device.py

```diff
@@ -103,7 +103,7 @@ def run(self, force=False):
                     result: Result = self.port.run_command(command)
                 except Exception as exception:  # pylint: disable=W0718
                     log.error("Error decoding result: %s", exception)
-                    result = Result(command_code=command.code, result_type=ResultType.ERROR, raw_response="")
+                    result = Result(command_code=command.code, result_type=ResultType.ERROR, raw_response=b"Error decoding result")
```

Comment:
this is where we should get more specific on error handling (eventually), eg PortNotConnected vs UnableToDecode type separation

Reply:
I agree we need work on the error handling, but lets save it for a future PR.

Reply:
agree